rpm/0004-Introduce-mount-sd-service-that-is-executed-as-user.patch

```diff
@@ -93,9 +93,9 @@ index 00000000..04a745d2
 +++ b/tools/udisksctl-user
 @@ -0,0 +1,4 @@
 +#!/bin/sh
-+ARGS=$@
-+DEVICEUSER=$(loginctl list-sessions | grep seat0 | tr -s " " | cut -d " " -f 4)
-+/bin/su -l $DEVICEUSER -c "/usr/bin/udisksctl $ARGS"
++ARGS="$@"
++DEVICEUSER="$(loginctl list-sessions | fgrep seat0 | tr -s ' ' | cut -d ' ' -f 4)"
```

Comment:
Don't use fgrep:
https://www.shellcheck.net/wiki/SC2197

Reply:
O.K., while this is only "problematic" in someone's head.

Reply:
The someone is upstream I think that is relevant.

Reply:
Well, I originally thought of the author of https://www.shellcheck.net/wiki/SC2197
The whole reasoning has been for decades (since ~1995) because "POSIX says so" and this made the GNU grep man-page also to point out `egrep` and `fgrep` as "deprecated" menawhile.  But it is just because "POSIX wants to say so", nothing else!  In their own words, "It was the clear intention of the standard developers to consolidate the three greps into a single command."  But this intention was doomed to fail right from the start.  `fgrep` and `egrep` will not go away, way too many scripts and script authors are using it.  And `grep -F` looks ugly.

Why do you think this is relevant?

Reply:
I won't go into meta type discussions here.

Reply:
So apparently you have no real reason to think of this as "problematic", except that you want to.